metricflow/model/transformations/boolean_measure.py

```diff
@@ -16,15 +16,15 @@ def transform_model(model: UserConfiguredModel) -> UserConfiguredModel:  # noqa:
             for measure in data_source.measures:
                 if measure.agg == AggregationType.BOOLEAN:
                     logger.warning(
-                        f"In data source {data_source.name}, measure `{measure.name}` is configured as "
+                        f"In data source {data_source.name}, measure `{measure.reference}` is configured as "
                         f"aggregation type `boolean`, which has been deprecated. Please use `sum_boolean` "
                         f"instead."
                     )
                 if measure.agg == AggregationType.BOOLEAN or measure.agg == AggregationType.SUM_BOOLEAN:
                     if measure.expr:
                         measure.expr = f"CASE WHEN {measure.expr} THEN 1 ELSE 0 END"
                     else:
-                        measure.expr = f"CASE WHEN {measure.name} THEN 1 ELSE 0 END"
+                        measure.expr = f"CASE WHEN {measure.reference} THEN 1 ELSE 0 END"
```

Comment:
And here `measure.reference.element_name`?

Reply:
Updated.